h/static/scripts/streamer.js

```diff
@@ -27,33 +20,18 @@ var socket;
  * @return The push notification service client.
  */
 // @ngInject
-function connect($rootScope, annotationMapper, groups, session, settings) {
-  // The public interface of the streamer. Returned from connect.
-  var controls = {
-    setConfig: setConfig
-  };
+function Streamer($rootScope, annotationMapper, groups, session, settings) {
+  // the randomly generated session UUID
+  var clientId = uuid.v4();
+
+  // the singleton socket instance, only one may
```

Comment:
Since `Streamer` is a class, which could not be instantiated multiple times, the word "singleton" can be removed.